imported/spirv/spvgen.h

```diff
@@ -421,10 +419,19 @@ bool InitSpvGen()
     }
 
     bool success = true;
+    const char* pLibName = SpvGeneratorName;
```

Comment:
pLibName -> pFullFileName

Reply:
Disagree with this misleading name. It is the full name only in the case that no directory was provided.

Reply:
OK. Please ignore this.